samples/core/src/main/java/io/oasp/gastronomy/restaurant/staffmanagement/batch/impl/staffimport/writer/StaffMemberWriter.java

```diff
@@ -0,0 +1,49 @@
+package io.oasp.gastronomy.restaurant.staffmanagement.batch.impl.staffimport.writer;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.item.ItemWriter;
+
+import io.oasp.gastronomy.restaurant.staffmanagement.logic.api.Staffmanagement;
+import io.oasp.gastronomy.restaurant.staffmanagement.logic.api.to.StaffMemberEto;
+
+/**
+ * StaffMemberWriter is responsible for writing StaffMemberEto to database.
+ *
+ * @author sroeger
+ */
+public class StaffMemberWriter implements ItemWriter<StaffMemberEto> {
+
+  private static final Logger LOG = LoggerFactory.getLogger(StaffMemberWriter.class);
+
+  private Staffmanagement staffManagement;
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
```

Comment:
remove unnecessary @inheritDoc javaDoc comments. They are simply not needed as @override is sufficient for the IDE.